tests/drawer/test_draw_catalyst.py

```diff
@@ -12,16 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Test the pennylane drawer with Catalyst."""
-# pylint: disable=import-outside-toplevel
+# pylint: disable=import-outside-toplevel,protected-access
 import pytest
 import pennylane as qml
 
 pyzx = pytest.importorskip("catalyst")
```

Comment:
we can stop importing catalyst in each test with this too:) would be nice to just use the `qml` integration, but I don't think it's ready on this branch yet so - for another rainy day.
```suggestion
catalyst = pytest.importorskip("catalyst")
```

Reply:
You shouldn't need to import Catalyst and these tests should pass as long as Catalyst is installed on the system.

Reply:
agreed for the future, but this branch doesn't have the `cond` integration yet, so the tests will fail if we use `qml.cond` instead of `catalyst.cond`. Once it's all on master, we can just do `_ = pytest.importorskip("catalsyt")` to verify its installation, then use `qml.cond`, `qml.qjit`, etc. but I won't worry about blocking this PR for that